jinja2/includes/polyfill_scripts.html

```diff
@@ -0,0 +1,14 @@
+<script>
+  window.fetch ||
+    document.write(
+      '<script src="https://unpkg.com/unfetch/polyfill"><\/script>'
```

Comment:
[`unfetch`](https://github.com/developit/unfetch) is a great `fetch` polyfill. It's the smallest one I know and has sufficient features. It's maintained and developed by Jason Miller @developit who started the Preact project amongst many other great libraries.